packages/block-editor/src/components/block-list/block.js

```diff
@@ -155,8 +155,7 @@ function BlockListBlock( {
 			'is-highlighted': isHighlighted,
 			'is-multi-selected': isMultiSelected,
 			'is-reusable': isReusableBlock( blockType ),
-			'is-dragging':
-				isDragging && ( isSelected || isPartOfMultiSelection ),
```

Comment:
What happened here? Why was `( isSelected || isPartOfMultiSelection )` needed?

Reply:
Before, `isDragging` was used to indicate whether any block is being dragged, and `isSelected || isPartOfMultiSelection` were added to the logic to identify the particular blocks being dragged. It's a little bit unstable since it relies on only being able to drag selected blocks.

Now the selector `isBlockBeingDragged` takes care of all of those things, which I think is a better way to do it.